tests/scripts/expect/_common.exp

```diff
@@ -58,6 +58,7 @@ proc wait_for {command success {failure {[\r\n]FAILURE_NOT_EXPECTED[\r\n]}}} {
 }
 
 proc expect_line {line} {
+    set timeout 10
```

Comment:
Since most calls to `expect_line` should return very quickly, the 10s timeout won't be causing any problem?

IIUC, we actually set the default timeout to 10s at the end of `_common.exp`